scripts/exceptions.py

```diff
@@ -0,0 +1,60 @@
+#Try except
+class B(Exception):
```

Comment:
Most of the code I am find is copied as it is from [python docs](https://docs.python.org/3/tutorial/errors.html#handling-exceptions)

Reply:
fix